runtime/common/src/weights/mod.rs

```diff
@@ -32,5 +32,4 @@ pub mod pallet_timestamp;
 pub mod pallet_treasury;
 pub mod pallet_utility;
 
-pub use block_weights::constants::BlockExecutionWeight;
-pub use extrinsic_weights::constants::ExtrinsicBaseWeight;
+pub use frame_support::weights::constants::{BlockExecutionWeight, ExtrinsicBaseWeight};
```

Comment:
Don't we want this to be our new weights?
- runtime/common/src/weights/block_weights.rs
- runtime/common/src/weights/extrinsic_weights.rs

```suggestion
pub mod block_weights;
pub mod extrinsic_weights;
```

Reply:
done